### PR TITLE
cluster/Vagrantfile: support CONTIV_K8S_VERSION of 1.7.x

### PR DESCRIPTION
**Pull request by dseevr**
Also, removed unused line

Signed-off-by: Bill Robinson <dseevr@users.noreply.github.com>

### CODE REVIEW

File: cluster/Vagrantfile [hunk 1]
```diff
@@ -30,7 +30,7 @@ node_os = ENV['CONTIV_NODE_OS'] || CENTOS
 k8s_ver = ENV['CONTIV_K8S_VERSION'] || DEFAULT_K8S_VERSION
 orc_path = case k8s_ver
            when /^v1\.[45]\./ then 'k8s1.4/'
-           when /^v1\.6\./    then 'k8s1.6/'
+           when /^v1\.[67]\./ then 'k8s1.6/'
```
Comment (gaurav-dalvi) [inline cluster/Vagrantfile]: Is the behavior / working of k8s 1.6 and 1.7 similar ? 
We are outputting to k8s.16 even if k8s version is 1.7.

**Comment by neelimamukiri** [comment]
@dseevr, have you tested this works with 1.7?

**Comment by dseevr** [comment]
@neelimamukiri I brought up a 1.7.0 cluster, created a network with netctl, loaded the `contiv-c1` and `contiv-c2` yamls from the docs (one pod on master, one pod on worker0), they can ping each other by IP and by hostname.

(I did manually restart the netplugins after demo-kubeadm came up because they were still in bridge mode)

File: cluster/Vagrantfile (hunk 1 above)
Comment (dseevr) [inline cluster/Vagrantfile]: Correct, we do the same thing with 1.4 and 1.5

**Pull request merged**